packages/desktop-client/src/components/table.tsx

```diff
@@ -722,7 +745,7 @@ export function SheetCell({
       }
       textAlign={textAlign}
       {...props}
-      value={sheetValue}
+      value={(sheetValue ?? '').toString()}
```

Comment:
Would this work here?
```suggestion
      value={String(sheetValue)}
```

Reply:
I think for `null` this would return 'null' instead of '', I think we probably want '' here?